roles/rac-db-setup/tasks/rac-db-install.yml

```diff
@@ -141,6 +141,9 @@
   become: yes
   become_user: "{{ oracle_user }}"
   command: "{{ oracle_home }}/OPatch/{{ item.method }} -silent -oh {{ oracle_home }} {{ swlib_unzip_path }}/{{ item.patchnum }}{{ item.patch_subdir }}"
+  environment:
+    PATH: "{{ grid_home }}/perl/bin:/usr/local/bin:/bin:/usr/bin:/usr/local/sbin:/usr/sbin"
```

Comment:
Should it be "{{ oracle_home }}/perl/bin":
```
    PATH: "{{ oracle_home }}/perl/bin:/usr/local/bin:/bin:/usr/bin:/usr/local/sbin:/usr/sbin"
```

Also, seems we are hard coding the remainder of the $PATH variables after the first path.
Can we collect the existing $PATH into an ansible variable (maybe in a separate task that precedes this) and then append it to the PATH env variable?

Reply:
Patch location fixed.  In terms of path references I'd prefer to follow the convention we already use in the toolkit code:

(for example)
https://github.com/google/bms-toolkit/blob/e1a7eba1e1241231d9831252a2251b8928dc0f0e/roles/patch/tasks/rac-dbpatch.yml#L28
https://github.com/google/bms-toolkit/blob/e1a7eba1e1241231d9831252a2251b8928dc0f0e/roles/dg-config/tasks/main.yml#L58
https://github.com/google/bms-toolkit/blob/41333ce0b588106f2c556f34636e898807ce5fd3/roles/db-create/tasks/rac-db-create.yml#L138